tests/FSharp.AWS.DynamoDB.Tests/Program.fs

```diff
@@ -212,6 +212,21 @@ let PaginationTests =
             testCase "Paginated Query with filter" paginationTests.``Paginated Query with filter``
         ]
 
+let metricsCollectorTestsTableFixture = new TableFixture()
+let metricsCollectorTests = ``Metrics Collector Tests``(metricsCollectorTestsTableFixture)
+let MetricsCollectorTests =
+    testList "MetricsCollectorTests"
+        [
+            testCase "Collect Metrics on GetItem" metricsCollectorTests.``Collect Metrics on GetItem``
```

Comment:
Can you add a link to the `TyrGet operation` test that I failed to add in #42 please? https://github.com/fsprojects/FSharp.AWS.DynamoDB/pull/42/files#diff-8357e3a808c62fd97a957d6c499bcb5237c7c4e4f1621164285d7d49dba73286R81

Reply:
will do, sorry I must have missed that as well

Reply:
Its almost the sort of thing that would trigger one to transition the codebase to use a boring test framework (say the word and I'll do a PR switch it back to Xunit)

Reply:
I spent about an hour today trying unsuccessfully to get it to run using `dotnet test` - I don’t have a problem with a different test framework, although I wouldn’t mind cutting a release before doing too many more changes.

Reply:
Sorry to hear that :( Will do a PR now and you can decide if/when to apply it

Reply:
https://github.com/fsprojects/FSharp.AWS.DynamoDB/pull/46 (as mentioned, does not need to go in now)